src/app/home/organizers/organizers.component.ts

```diff
@@ -1,91 +1,15 @@
 import { Component, OnInit, ViewChild } from '@angular/core';
-
-import { ModalDirective } from 'ngx-bootstrap/modal';
-import { timer } from 'rxjs';
-
-import { Organizer } from '../../core/models/organizer.model';
+import organizers from 'src/assets/data/organizers';
```

Comment:
Sort imports. The way Angular used to suggest import order is the following one:
- Angular libraries
- External libraries
- Internal imports

This would go something like this:
```
import { Component, OnInit } from '@angular/core';

import organizers from '../../../assets/data/organizers';
```

Also, ViewChild is not used anymore.